### ISSUE TITLE
Cannot open another gui from the first gui opened by the player

### ISSUE THREAD

**Issue by AraaCuteUwU** [Muqsit/InvMenu#243]

**Comment by xKaOz** [comment]
A bit more context on this, I also have a similar issue, when I was using a DoubleChest as the menu it wouldn’t send very often, I tried a few different things, not sure how this works but I moved my code over to be a Hopper, and it sends more regularly? But after running the command a few times it just stops working, and I don’t really see why.

Before people ask yes i close the menu properly.

I’ll attach a video below, if you want to see some code about how I send the menu ect, just ask.

(https://streamable.com/j5vusa) the error you see in game is one I’m sending to the player inside $menu->send(…)

**Comment by AraaCuteUwU** [comment]
> A bit more context on this, I also have a similar issue, when I was using a DoubleChest as the menu it wouldn’t send very often, I tried a few different things, not sure how this works but I moved my code over to be a Hopper, and it sends more regularly? But after running the command a few times it just stops working, and I don’t really see why.
> 
> Before people ask yes i close the menu properly.
> 
> I’ll attach a video below, if you want to see some code about how I send the menu ect, just ask.
> 
> (https://streamable.com/j5vusa) the error you see in game is one I’m sending to the player inside $menu->send(…)

May I see the code?

**Comment by xKaOz** [comment]
At this point my code is useless, i tested with a blank function to send a empty UI, on the latest commit, and it only sent twice.

![image](https://github.com/Muqsit/InvMenu/assets/83518772/b84d1b32-4300-4d6a-886d-098876858d8a)

I randomly tried to test a old commit (https://github.com/Muqsit/InvMenu/tree/706e307d705ef8fe20653bebd04a18b1d9b3e47) and everything is now working fine, with my original menu.

But honestly i don't see what could be wrong with his latest commit, what would've caused this.

**Comment by AraaCuteUwU** [comment]
> At this point my code is useless, i tested with a blank function to send a empty UI, on the latest commit, and it only sent twice.
> 
> ![image](https://private-user-images.githubusercontent.com/83518772/320085583-b84d1b32-4300-4d6a-886d-098876858d8a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIzNDcwMTYsIm5iZiI6MTcxMjM0NjcxNiwicGF0aCI6Ii84MzUxODc3Mi8zMjAwODU1ODMtYjg0ZDFiMzItNDMwMC00ZDZhLTg4NmQtMDk4ODc2ODU4ZDhhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA1VDE5NTE1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIyMTJhYzAyMjQwY2Y2MWQ5ZjNjNGU4ODQ2NWQzMDk1ZGJkMDIzMjNkODlmMDkzNmIyYTljZjdmNjE5MzFhMGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Tgn-OOYrYF3hp52APgtOfTV9qwbwK_4_gffiQDronmU)
> 
> I randomly tried to test a old commit (https://github.com/Muqsit/InvMenu/tree/706e307d705ef8fe20653bebd04a18b1d9b3e47) and everything is now working fine, with my original menu.
> 
> But honestly i don't see what could be wrong with his latest commit, what would've caused this.

I don't think there's anything wrong with my code, my code is the same as that

**Comment by xKaOz** [comment]
@AraaCuteUwU  no, thats not what i was saying i mean they is a issue with InvMenu related to the latest commit, you can use the older commit ((https://github.com/Muqsit/InvMenu/tree/706e307d705ef8fe20653bebd04a18b1d9b3e47) like i did, to solve this.
